source/index.html.md.erb

```diff
@@ -21,6 +21,11 @@ Welcome to the technical documentation for apps and libraries in Teacher Service
 | Register trainee teachers | [DFE-Digital/register-trainee-teachers](https://github.com/DFE-Digital/register-trainee-teachers) | Publish & Register |
 | Teacher Training API | [DFE-Digital/teacher-training-api](https://github.com/DFE-Digital/teacher-training-api) | Publish & Register |
 | Teaching vacancies | [DFE-Digital/teaching-vacancies](https://github.com/DFE-Digital/teaching-vacancies) | Teaching vacancies |
+| Get Into Teaching Application | [DFE-Digital/get-into-teaching-app](https://github.com/DFE-Digital/get-into-teaching-app) | Get Into Teaching |
+| Get Into Teaching API | [DFE-Digital/get-into-teaching-api](https://github.com/DFE-Digital/get-into-teaching-api) | Get Into Teaching |
+| Get Teacher Training Adviser | [DFE-Digital/get-teacher-training-adviser-service](https://github.com/DFE-Digital/get-teacher-training-adviser-service) | Get Into Teaching |
+| Get Into Teaching Asset Manager | [DFE-Digital/GITISContent](https://github.com/DFE-Digital/GITISContent) | Get Into Teaching |
+| Get School Experience | [DFE-Digital/schools-experience](https://github.com/DFE-Digital/schools-experience) | School Experience |
```

Comment:
as above

Reply:
Those are coming from the docs pages in the repos, as I mentioned in the comments I will need to change them all, but that would have to be a PR in that repo rather than here.
Since they have all been 'bodged' in the past its an opportunity to fix them